macOS-HS-libvirt.xml

```diff
@@ -66,18 +61,16 @@
     <feature policy='require' name='xsave'/>
     <feature policy='require' name='avx'/>
     <feature policy='require' name='xsaveopt'/>
-    <feature policy='require' name='avx2'/>
     <feature policy='require' name='smep'/>
   </cpu>
-<!-- Note: To resolve clock issues experienced on Fedora 27 as described at top of file,
-     we need to add a timer. Replace the clock offset line with,
-
-     <clock offset='utc'>
-       <timer name='tsc'/>
-     </clock>
-
-     Please remember to also change the kvm hidden state as well, as described above.
--->
+  <!-- Note: The frequency is required for KVMs "Suspend to Disk" feature and
+       must match your host CPUs TSC frequency (in Hz).
+       You can obtain the TSC frequency under Linux with "dmesg | grep -i tsc".
+       When you don't need "Suspend to Disk" delete the frequency attribute.
+  -->
+  <clock offset='utc'>
+    <timer name='tsc' frequency='3400022000'/>
```

Comment:
Where does this hardcoded frequency come from? It can't be right for everybody, or is it?

Reply:
See the comment above. The value must match your Host CPU.
I simply inserted the value I currently use, shall I replace it with something else?